docker/Dockerfile

```diff
@@ -202,18 +202,17 @@ RUN if [ -z "$PYTORCH_NIGHTLY_URL" ] ; then \
 	        torchvision==${TORCHVISION_VERSION}.${PYTORCH_NIGHTLY_VERSION} ; \
     fi
 
-#####################################
-# Install EFA and AWS-OFI-NCCL plugin
-#####################################
+#############
+# Install EFA
+#############
 
-ARG EFA_INSTALLER_VERSION=1.38.1
-ARG AWS_OFI_NCCL_VERSION
+ARG EFA_INSTALLER_VERSION
 
 ENV LD_LIBRARY_PATH=/usr/local/cuda/extras/CUPTI/lib64:/opt/amazon/openmpi/lib:/opt/amazon/efa/lib:/opt/aws-ofi-nccl/install/lib:$LD_LIBRARY_PATH
```

Comment:
Did you mean to update the LD_LIBRARY_PATH to drop `/opt/aws-ofi-nccl/install/lib` in favor of the path from the installer as well, given you dropped the custom build now? Ref [doc](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-start-nccl.html#nccl-start-base-enable), it is in `/opt/amazon/ofi-nccl`

Reply:
ah yes, thanks for the catch!